ACKategoriesCore/ACKUrl.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ACKUrl.swift
+//  ACKategories
+//
+//  Created by Alexandr Grigoryev on 21.02.2023.
+//
+
+import Foundation
+
+public struct ACKUrl: RawRepresentable {
```

Comment:
🙏 please rename it to `BetterURL`

Reply:
Move the `RawRepresentable` implementation to the `extension`. With that with can initialize `BetterURL` with normal `URL`

Reply:
I think there is no need to move it to extension as if you move it, the compiler generated initializer would be internal, we need it public so instead of moving it to extension, I would add additional public initializer that takes `URL`